src/Stratis.FederatedPeg.Features.FederationGateway/Interfaces/ICrossChainTransfer.cs

```diff
@@ -31,6 +30,11 @@ public interface ICrossChainTransfer : IBitcoinSerializable
         /// </summary>
         long DepositAmount { get; }
 
+        /// <summary>
+        /// The chain A deposit height of the transaction. Is null if only seen in a block.
```

Comment:
Not sure about the comment, if seen in a block, then you should have the block height ? I had understood it could be null if it is pending (ie received from a peer but not matched on chain A yet due if chain A is being synced)

Reply:
See below.

Reply:
Ah OK sorry I am being stupid